monitor.py

```diff
@@ -118,9 +118,14 @@ def __init__(self):
 
     def start(self, root):
         self.root = root
-        logdir = expanduser(config.get('journaldir') or config.default_journal_dir)  # type: ignore # config is weird
+        journal_dir = config.get('journaldir') or config.default_journal_dir
 
-        if not logdir or not isdir(logdir):  # type: ignore # config does weird things in its get
+        if journal_dir is None:
+            journal_dir = ''
+
+        logdir = expanduser(journal_dir)  # type: ignore # config is weird
```

Comment:
The comment belongs above ;)

Reply:
Has to be on the same line to make it ignored by static type checkers, as do all "old style" type comments